node-graph/gstd/src/quantization.rs

```diff
@@ -0,0 +1,93 @@
+use graphene_core::raster::{Color, Image};
+use graphene_core::Node;
+
+pub struct GenerateQuantizationNode<N: Node<(), Output = u32>, M: Node<(), Output = u32>>(N, M);
```

Comment:
It would be great if you could write a comment explaining what quantization is and what it achieves.